geode-for-redis/src/main/java/org/apache/geode/redis/internal/commands/executor/server/CommandCommandExecutor.java

```diff
@@ -77,6 +77,10 @@ private RedisResponse executeNoSubcommand(ExecutionHandlerContext context) {
       oneCommand.add(type.lastKey());
       oneCommand.add(type.step());
 
+      List<String> categories = new ArrayList<>();
+      categories.add("@" + type.category().name().toLowerCase());
```

Comment:
could you use Collections.singletonList here?

Reply:
Fixed.